tracer/src/Datadog.Trace.ClrProfiler.Native/integration.h

```diff
@@ -16,6 +17,11 @@ namespace trace
 
 const size_t kPublicKeySize = 8;
 const shared::WSTRING tracemethodintegration_assemblyname = WStr("#TraceMethodFeature");
+const std::unordered_set<shared::WSTRING> tracemethodintegration_wildcard_ignored_methods(
```

Comment:
niche, but wondering if we should add `Finalize` to this list?  This has implications if they _do_ have a method called `Finalize` which _isn't_ a finalizer though, so I'm not sure whether it's a good idea or not.

Reply:
In theory we can tell the difference by checking if `Finalize()` overrides `object.Finalize()`, but even without that, I agree we should ignore it by default.

Reply:
In the latest commits I added `Finalize` to the list of method names to ignore by default, including ones that aren't finalizers. I don't think this will be an issue for now and we can revisit this later if it becomes an issue